src/SeqCli/Cli/Commands/Forwarder/InstallCommand.cs

```diff
@@ -29,234 +27,127 @@
 
 // ReSharper disable once ClassNeverInstantiated.Global
 
-namespace SeqCli.Cli.Commands.Forwarder
-{
-    [Command("forwarder", "install", "Install the forwarder as a Windows service", IsPreview = true)]
-    [SuppressMessage("Interoperability", "CA1416:Validate platform compatibility")]
-    class InstallCommand : Command
-    {
-        readonly StoragePathFeature _storagePath;
-        readonly ServiceCredentialsFeature _serviceCredentials;
-        readonly ListenUriFeature _listenUri;
-        
-        bool _setup;
+namespace SeqCli.Cli.Commands.Forwarder;
 
-        public InstallCommand()
-        {
-            _storagePath = Enable<StoragePathFeature>();
-            _listenUri = Enable<ListenUriFeature>();
-            _serviceCredentials = Enable<ServiceCredentialsFeature>();
+[Command("forwarder", "install", "Install the forwarder as a Windows service", IsPreview = true)]
+[SuppressMessage("Interoperability", "CA1416:Validate platform compatibility")]
+class InstallCommand : Command
+{
+    readonly StoragePathFeature _storagePath;
+    readonly ServiceCredentialsFeature _serviceCredentials;
+    readonly ListenUriFeature _listenUri;
 
-            Options.Add(
-                "setup",
-                "Install and start the service only if it does not exist; otherwise reconfigure the binary location",
-                _ => _setup = true);
```

Comment:
The Seq codebase, an earlier version of which this came from, includes a `--setup` flag on its service install command to handle reconfiguration during MSI upgrades, in case the user chose a different path to install the Seq binaries.

`seqcli forwarder` doesn't need that - it's sufficient to be able to `forwarder uninstall` and `forwarder install` at the command line, if the `seqcli` binaries move.